src/program/lwaftr/setup.lua

```diff
@@ -204,6 +204,21 @@ local function link_sink(c, v4_out, v6_out)
    config.link(c, 'fragmenterv6.output -> '..v6_out)
 end
 
+function load_iface(c, conf, v4_nic_name, v6_nic_name)
```

Comment:
Can we add "kernel" or "linux" to the name?  Otherwise I think I will be confused.

In some ideal world, the physical NIC drivers and the kernel drivers should have a similar interface so that we can avoid code duplication!